test/es6module/bug_issue_5777.js

```diff
@@ -0,0 +1,24 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+
+// Bug Issue 5777 https://github.com/Microsoft/ChakraCore/issues/5777
+// Duplicate export names should cause an early syntax error
+
+WScript.RegisterModuleSource("a.js",
+    `export const boo = 4;
+    export {bar as boo} from "b.js";
+    print ("Should not be printed")`);
+WScript.RegisterModuleSource("b.js","export const bar = 5;");
+
+import("a.js").then(()=>{
+    print("Failed - expected error not thrown")
+}).catch ((e)=>{
+    const errorString = e.constructor.toString();
+    if (errorString === SyntaxError.toString()) {
+        print("pass");
```

Comment:
Does `e.constructor === SyntaxError` not work? I would have thought they'd be reference-equal, as there shouldnt be multiple instances of the SyntaxError constructor.

Reply:
This wasn't necessary - fixed.

Reply:
There could be multiple instances only if you had more than one JsContext—not likely to happen in the test environment.